### ISSUE TITLE
Example of CustomCell

### ISSUE THREAD

**Issue by kirillplatonov** [aksonov/react-native-tableview#17]
Hi guys. Amazing job!

Could you please add example of creating and using custom cell to readme?

**Comment by kirillplatonov** [comment]
I tried to follow this example https://github.com/aksonov/react-native-tableview/blob/master/examples/TableViewDemo/index.ios.js
But when I add custom cell to my table view I receive this errors in console: https://www.dropbox.com/s/bunsgugrhwi1kni/Screenshot%202015-10-04%2010.58.28.png?dl=0

My code:
```js
'use strict';

var React = require('react-native');
var {
  StyleSheet,
  Text,
  View,
  TouchableHighlight,
  TextInput,
  ScrollView,
} = React;
var TableView = require('react-native-tableview');
var Section = TableView.Section;
var Item = TableView.Item;
var Cell = TableView.Cell;

var indexScene = React.createClass({
  render: function() {
    return (
        <TableView
          style={s.table}
          tableViewStyle={TableView.Consts.Style.Grouped}
          tableViewCellStyle={TableView.Consts.CellStyle.Value1}
          onPress={(event) => console.log(event)}>
          <Section arrow={true}>
            <Cell><Text>Cell 3</Text></Cell>
          </Section>
        </TableView>
    );
  }
});

var s = StyleSheet.create({
  table: {
    flex: 1,
  },
});

module.exports = indexScene;
```

Maybe someone has similar problem and know how to fix this?

**Comment by kirillplatonov** [comment]
I tried version 1.2.4 and it working there properly. So problem caused by this commit https://github.com/aksonov/react-native-tableview/commit/a701d84354b3b9988370423af18db573795f6cd2

**Comment by aksonov** [comment]
Should be fixed with 1.3.1
